.travis.yml

```diff
@@ -1,23 +1,27 @@
 language: php
 php:
   - 5.4
+env:
+  - GDT_DRUPAL_CORE="7" GDT_TEST_URL="http://127.0.0.1:8080/misc/drupal.js"
```

Comment:
We already have GRUNT_DRUPAL_QUIET, we should probably align env variable prefixes. We could make GRUNT_DRUPAL_ deprecated and use GDT_ going forward.